openlibrary/catalog/works/by_author.py

```diff
@@ -7,7 +7,7 @@
 from openlibrary.catalog.utils import cmp, mk_norm
 from openlibrary.catalog.read_rc import read_rc
 from collections import defaultdict
-from pprint import pformat
+
 from catalog.utils.edit import fix_edition
 import urllib
 sys.path.append('/home/edward/src/olapi')
```

Comment:
User-specific hardcoded path

Reply:
Please remove line 13.

Reply:
Presumably this code won't work without appending @EdwardBetts's `olapi` I'm going to be unpopular and suggest it be left so/until we can figure out what this was supposed to be